res/Sectors/M1105/Porlock.txt

```diff
@@ -0,0 +1,446 @@
+# Generated by https://travellermap.com
+# 2018-11-05T12:57:58+00:00
+
+# Porlock
+# 5,4
+
+# Name: Porlock (Hv)
+
+# Milieu: M1105
+
+# Credits: Porlock sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Deeskor
+# Subsector B: Puaska
+# Subsector C: Pycran
+# Subsector D: Eaneneguah
+# Subsector E: Gledoxkinixvoor
+# Subsector F: Gekelgotrox
+# Subsector G: Kanblophlare
+# Subsector H: Phleteire
+# Subsector I: Slinqun
+# Subsector J: Rirekluan
+# Subsector K: Muackarap
+# Subsector L: Goornthieqnan
+# Subsector M: Arneetet
+# Subsector N: Ad Eqerkax
+# Subsector O: Anpaan
+# Subsector P: Goophl
+
+# Alleg: CsHv: "Hive Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"

Reply:
Done